packages/telemed-ehr/app/src/telemed/utils/appointment-accessibility.helper.ts

```diff
@@ -4,24 +4,27 @@ import {
   allLicensesForPractitioner,
   ApptStatus,
   checkIsEncounterForPractitioner,
+  getStatusFromExtension,
   mapStatusToTelemed,
   PractitionerLicense,
   PractitionerQualificationCode,
+  VisitStatus,
 } from 'ehr-utils';
 
 export type GetAppointmentAccessibilityDataProps = {
   location?: Location;
   encounter: Encounter;
   appointment?: Appointment;
   user?: OttehrUser;
+  appointmentType?: string;
 };
 
 export type GetAppointmentAccessibilityDataResult = {
   allLicenses?: PractitionerLicense[];
   availableStates?: string[];
   state?: string;
   isStateAvailable: boolean;
-  status?: ApptStatus;
+  status?: ApptStatus | VisitStatus | undefined;
```

Comment:
are ApptStatus and VisitStatus different?

Reply:
yes

Reply:
How are they different?

Reply:
ApptStatus is for telemed appointment statuses and VisitStatus is for in-person appointment statuses.

```
export enum ApptStatus {
  'ready' = 'ready',
  'pre-video' = 'pre-video',
  'on-video' = 'on-video',
  'unsigned' = 'unsigned',
  'complete' = 'complete',
  'cancelled' = 'cancelled',
}
```

```
const STATI = [
  'ARRIVED',
  'CANCELLED',
  'CHECKED-OUT',
  'INTAKE',
  'NO-SHOW',
  'PENDING',
  'PROVIDER',
  'READY',
  'DISCHARGE',
  'PROVIDER-READY',
];
type VISIT_STATUS_LABEL_TYPE = typeof STATI;
export type VisitStatus = VISIT_STATUS_LABEL_TYPE[number];
```

Reply:
Can you rename them please

Reply:
Can you rename them please?

Reply:
yes https://github.com/masslight/ottehr/issues/575